src/main/java/racingcar/domain/RacingCircuit.java

```diff
@@ -0,0 +1,62 @@
+package racingcar.domain;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.IntStream;
+import racingcar.generator.NumberGenerator;
+
+public class RacingCircuit {
+
+    private final List<Car> cars = new ArrayList<>();
+    private final NumberGenerator generator;
+
+    public RacingCircuit(final NumberGenerator generator) {
+        this.generator = generator;
+    }
+
+    public void registerCars(final List<String> carNames) {
+        cars.addAll(carNames.stream()
+                .filter(Objects::nonNull)
+                .map(Car::new)
+                .toList());
+    }
+
+    public List<MoveResults> startRace(final int raceTryCount) {
+        validateRaceTryCount(raceTryCount);
+
+        return IntStream.range(0, raceTryCount)
+                .mapToObj(i -> moveCars())
+                .toList();
+    }
+
+    private void validateRaceTryCount(final int raceTryCount) {
+        if (raceTryCount <= 0) {
```

Comment:
```suggestion
        if (raceTryCount < 1) {
```
로 변경하고 1을 의미있는 상수로 변경한다음 메시지에도 이 상수를 활용하면 좋을 것 같네요.

Reply:
와,,, 맞네요 🥲